src/app/view/view.component.ts

```diff
@@ -181,10 +181,10 @@ export class ViewComponent implements OnInit, OnDestroy {
     const title = null;
     const message = this.i18n({
       id: 'view.dialog.clone.message',
-      value: 'Do you want to create a copy of the view or just rename?',
+      value: 'Do you want to create a copy of the view or just update existing?',
```

Comment:
I think it should be `update the existing one` in order to sense in English.

Reply:
Thanks. Resolved